src/pages/Cart/Cart.jsx

```diff
@@ -77,13 +78,16 @@ function Cart() {
             </div>
           </div>
         )}
+      </section>
         {Cart.length === 0 && (
-          <div>
-            <h1>Hey,it feels so light!</h1>
-            <p className="txt-sm">There is nothing in your bag</p>
-          </div>
+           <div className="flex empty-page-box" >
+           <h1 className="empty-page-title" >YOUR CART IS EMPTY !</h1>
+           <img className="flex empty-page-image" src="https://images.bewakoof.com/images/doodles/empty-cart-page-doodle.png" alt="empty cart" />
+           <Link to="/shop">
+           <button className="btn btn-sm outline ">View Products</button>
+           </Link>
+         </div>
```

Comment:
appreciation - good that you're showing a page for empty cart

Reply:
thanks